src/adaptors/tcp/tcp_adaptor.c

```diff
@@ -1736,13 +1732,26 @@ static void connection_run_LSIDE_IO(qd_tcp_connection_t *conn)
             //
             // Don't do anything
             //
+            return;
             break;
```

Comment:
the "break" is dead code - it can be removed to prevent test tools from flagging it.